src/Tools/dotnet-monitor/Exceptions/ExceptionsOperation.cs

```diff
@@ -96,6 +98,14 @@ await using (Utf8JsonWriter writer = new(stream, new JsonWriterOptions() { Inden
                 writer.WriteString("typeName", instance.TypeName);
                 writer.WriteString("moduleName", instance.ModuleName);
                 writer.WriteString("message", instance.Message);
+
+                MemoryStream outputStream = new();
+                await JsonSerializer.SerializeAsync(outputStream, instance.CallStackResult, cancellationToken: token);
+                outputStream.Position = 0;
+
+                writer.WritePropertyName("callStack");
+                writer.WriteRawValue(new StreamReader(outputStream).ReadToEnd());
```

Comment:
Won't this write the entire call stack payload on a single line without white space? That's not desirable here.

I understand that we use `JsonSerializer` [here](https://github.com/dotnet/dotnet-monitor/blob/main/src/Microsoft.Diagnostics.Monitoring.WebApi/Stacks/JsonStacksFormatter.cs#L69) but we should probably use the existing `Utf8JsonWriter` instead.

You can at least share the property names since those are declared as constants on the model attributes, however it might be best to change the stacks feature to use `Utf8JsonWriter` instead of `JsonSerializer`, create a common writing method, and use the method in both features.